src/components/application_manager/CMakeLists.txt

```diff
@@ -404,6 +414,13 @@ set(LIBRARIES
   emhashmap -L${EMHASHMAP_LIBS_DIRECTORY}
 )
 
+if(REMOTE_CONTROL)
+    SET (LIBRARIES
+      ${LIBRARIES}
+      FunctionalModule
```

Comment:
@JackLivio functional module is component that is able to load plugins for RPC processing. RemoteControl is one of this plugging